### PR TITLE
Add operator tab in `network_monitoring/performance/setup.md`

### PR DESCRIPTION
**Pull request by kei6u**
### What does this PR do?

Add operator tab in `network_monitoring/performance/setup.md`.

### Motivation

There is no instruction of using operator in `network_monitoring/performance/setup.md`.

### Preview

https://github.com/kei6u/documentation/blob/kei6u/add_operator_tab_in_network_monitoring_performance/content/en/network_monitoring/performance/setup.md

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by alai97** [comment]
Created editorial review ticket (DOCS-2806).

File: content/en/network_monitoring/performance/setup.md [hunk 1]
```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/setup.md:347]:
```suggestion
{{% tab "Operator" %}}
<div class="alert alert-warning">The Datadog Operator is in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>

```

**Comment by kayayarai** [comment]
I'm getting this reviewed by the Operator and NPM folks. ...

File: content/en/network_monitoring/performance/setup.md [hunk 2]
```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
...
+  features:
+    networkMonitoring:
+      enabled: true
+  agent:
+    # (...)
+    systemProbe:
+      enabled: true
+```
+
+2. Add the following volumes to your manifest:
```
Comment (brycekahle) [inline content/en/network_monitoring/performance/setup.md:369]: I reviewed the operator and I don't think step 2 is necessary. It should be done for you. https://github.com/DataDog/datadog-operator/blob/1953406f4fddfef3bccc4e93a96193b71eacfb0c/controllers/datadogagent/utils.go#L1095-L1109

File: content/en/network_monitoring/performance/setup.md [hunk 3]
```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
...
+  namespace: placeholder
+spec:
+  # (...)
+  features:
+    networkMonitoring:
+      enabled: true
+  agent:
+    # (...)
+    systemProbe:
+      enabled: true
```
Comment (brycekahle) [inline content/en/network_monitoring/performance/setup.md:366]: I think you also need to set for NPM to work properly:
```
agent:
  process:
    enabled: true
```

File: content/en/network_monitoring/performance/setup.md [hunk 4]
```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
+[The Datadog Operator][1] is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
```
Comment (brycekahle) [inline content/en/network_monitoring/performance/setup.md:348]: This should include the same beta warning as https://docs.datadoghq.com/agent/kubernetes/?tab=operator#deploy-an-agent-with-the-operator

File: content/en/network_monitoring/performance/setup.md (hunk 3 above)
Comment (clamoriniere) [reply content/en/network_monitoring/performance/setup.md:366]: It should not be the case.

with:
```yaml
 features:
    networkMonitoring:
      enabled: true
```
the system-probe container will be enable automatically.

File: content/en/network_monitoring/performance/setup.md (hunk 2 above)
Comment (clamoriniere) [reply content/en/network_monitoring/performance/setup.md:369]: Agree

File: content/en/network_monitoring/performance/setup.md (hunk 3 above)
Comment (brycekahle) [reply content/en/network_monitoring/performance/setup.md:366]: @clamoriniere is correct. There is some glue code that makes this work. You only need what he said.

**Review by clamoriniere — APPROVED** [review]

File: content/en/network_monitoring/performance/setup.md [hunk 5]
```diff
@@ -344,6 +344,27 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
+<div class="alert alert-warning">The Datadog Operator is in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>
```
Comment (kayayarai) [inline content/en/network_monitoring/performance/setup.md:348]:
```suggestion
<div class="alert alert-warning">The Datadog Operator is in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>

```

Comment (kayayarai) [reply content/en/network_monitoring/performance/setup.md:348]: Formatting of the link below is broken without this newline

**Review by kayayarai — APPROVED** [review]

**Review by brycekahle — APPROVED** [review]

**Pull request merged**